.golangci.yml

```diff
@@ -72,3 +73,12 @@ issues:
     # and in most cases it's ok. This is copied from the golangci-lint defaults
     # mut modified to include go-kit logging.
     - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv|.*\.Log). is not checked
+
+linters-settings:
+  depguard:
+    list-type: denylist
+    include-go-root: true
+    packages-with-error-message:
```

Comment:
Maybe we want to add one for golang.org/x/sync/errgroup to use github.com/oklog/run instead?